whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/client/NrtmRestClient.java

```diff
@@ -79,17 +79,17 @@ public List<String> getNrtmAvailableSources(){
         }
     }
 
-    public UpdateNotificationFileResponse getNotificationFile(final String source){
+    public String getNotificationFileSignature(final String source){
         return client.target(String.format("%s/%s", baseUrl, source))
-                .path("update-notification-file.json")
+                .path("update-notification-file.jose")
                 .request(MediaType.APPLICATION_JSON_TYPE)
```

Comment:
I think media type is application-jose. Can you check ?

Reply:
yes, I checked the ITs already. It is JSON

Reply:
I just double checked and it is "jose+json"